src/EditorFeatures/Core.Wpf/Options/LegacyEditorConfigDocumentOptionsProvider.cs

```diff
@@ -135,6 +149,21 @@ private void ReleaseContext_NoLock(Task<ICodingConventionContext> contextTask)
 
         public async Task<IDocumentOptions> GetOptionsForDocumentAsync(Document document, CancellationToken cancellationToken)
         {
+            if (_enabled is null)
+            {
+                var experimentationService = await _experimentationServiceFactory.GetExperimentationServiceAsync(cancellationToken).ConfigureAwait(false);
+                _enabled = !EditorConfigDocumentOptionsProviderFactory.ShouldUseNativeEditorConfigSupport(experimentationService);
+                if (_enabled is false)
+                {
+                    UnregisterListeners();
+                }
+            }
+
+            if (_enabled != true)
```

Comment:
```suggestion
            if (_enabled is true)
```

Since the null shouldn't have gotten through at this point?

Reply:
This reads a bit strange, but the alternatives so far don't seem to be "obviously better" so I'll keep it this way for now to minimize churn.